x-pack/plugins/maps/common/constants.ts

```diff
@@ -26,6 +26,7 @@ export const EMS_TILES_VECTOR_TILE_PATH = 'vector/tile';
 
 export const MAP_SAVED_OBJECT_TYPE = 'map';
 export const APP_ID = 'maps';
+export const APP_ID_OSS = 'mapsOss';
```

Comment:
Should this be imported from maps_oss?

Reply:
AS I have mentioned above the reason I didn't do it is that I wanted to avoid to link the two plugins only for one constant.

Reply:
I have made this change